scripts/load_cof_r2.py

```diff
@@ -51,87 +50,93 @@ def create_sections(path_prefix, round_id, forms_config):
-                    " express your interest in the fund</a>."
-                ),
-            },
-            {
-                "id": "5cf439bf-ef6f-431e-92c5-a1d90a4dd32f",
-                "title": "Round 2 Window 3",
-                "short_name": "R2W3",
-                "opens": "2022-10-04 12:00:00",
-                "deadline": "2022-12-14 11:59:00",
-                "fund_id": "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4",
-                "assessment_deadline": "2023-03-30 12:00:00",
-                "prospectus": "https://www.gov.uk/government/publications/community-ownership-fund-prospectus",
-                "privacy_notice": "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice",
-                "contact_email": "COF@levellingup.gov.uk",
-                "contact_phone": None,
-                "contact_textphone": None,
-                "support_times": "9am to 5pm",
-                "support_days": "Monday to Friday",
-                "instructions": (
-                    "You must have received an invitation to apply. If we did not"
-                    " invite you, first <a"
-                    ' href="https://www.gov.uk/government/publications/community-ownership-fund-prospectus">'
-                    " express your interest in the fund</a>."
-                ),
-            },
-        ]
+    # Do we want to reuse the application sections config?
 
-        inserted_fund = insert_fund_data(fund_config)
-        print("Fund inserted:")
-        print(inserted_fund)
-        inserted_rounds = insert_round_data(rounds_config)
-        print("Rounds inserted:")
-        print(inserted_rounds)
+    # Separate config for r2w2 and r2w3
+    # create_sections("1", cof_form_config.COF_ROUND_2_ID, cof_form_config.COF_R2_ORDERED_FORMS_CONFIG, True)
+    # create_sections("2", cof_form_config.COF_ROUND_2_W3_ID, cof_form_config.COF_R2_ORDERED_FORMS_CONFIG, True)
 
-        # Do we want to reuse the application sections config?
+    # reuse config between r2w2 and r2w3
+    create_sections(
+        "1",
+        cof_form_config.COF_ROUND_2_ID,
+        cof_form_config.COF_R2_ORDERED_FORMS_CONFIG,
+    )
+    create_sections("1", cof_form_config.COF_ROUND_2_W3_ID, None)
 
-        # Separate config for r2w2 and r2w3
-        # create_sections("1", cof_form_config.COF_ROUND_2_ID, cof_form_config.COF_R2_ORDERED_FORMS_CONFIG, True)
-        # create_sections("2", cof_form_config.COF_ROUND_2_W3_ID, cof_form_config.COF_R2_ORDERED_FORMS_CONFIG, True)
 
-        # reuse config between r2w2 and r2w3
-        create_sections(
-            "1",
-            cof_form_config.COF_ROUND_2_ID,
-            cof_form_config.COF_R2_ORDERED_FORMS_CONFIG,
-        )
-        create_sections("1", cof_form_config.COF_ROUND_2_W3_ID, None)
+if __name__ == "__main__":
+    from app import app
```

Comment:
Do we need?

```python
import sys; sys.path.insert(1, ".");
```

Reply:
Not required. I have verified this command again `docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r3w1` it is working fine without import errors.